docusaurus/docs/React/custom-code-examples/channel-users-list.mdx

```diff
@@ -0,0 +1,123 @@
+---
+id: channel-user-lists
+sidebar_position: 21
+title: Channel Members and Online Status
+---
+
+In this example, we will demonstrate how to render the current channel members and their online status.
+
+## Render the Channel Members List
+
+Let's start the example by creating a simple members list component. To access the members list of the current channel, we will get the current channel using `useChannelStateContext` hook. 
+The example is a bit more convoluted, since we will add online presence updates at the next step.
+
+:::note
+In order for the client to receive updates for user presence, ensure that you are watching the channel with `channel.watch({ presence: true })`. More details can be found in the [&nearr; LLC documentation](https://getstream.io/chat/docs/javascript/watch_channel/?language=javascript).
```

Comment:
Struggling to marry the linked documentation with this example. Is there anyway this example could include the example configuration with `presence: true`?